aiutils/tftools/placeholder_management.py

```diff
@@ -49,9 +51,16 @@ def add_placeholder(self, name, dtype, shape=None, sparse=False):
             name (str): Name of the placeholder.
             dtype (tf.Dtype): Data type for the placeholder.
             shape (list of ints): Shape of the placeholder.
+            size (int) : If not None a list of placeholders is created with
+                the size being the length of the list. plh[name] returns the 
+                list of placeholders in this case.
             sparse (bool): Specifies if the placeholder takes sparse inputs.
         """
 
+        if size:
```

Comment:
I am using it to identify if plh['name'] returns list of placeholders or just a single placeholder. The idea is that you should get back what you feed in. It could be handled in code and size=1 list could  be used for individual placeholders but would require significant restructuring of code.